config/common/buttons.yaml

```diff
@@ -30,7 +30,17 @@ event:
       - triple_press
       - long_press
 
+
 binary_sensor:
+  - platform: gpio
+    id: line_out_sensor
+    pin:
+      pcm5122: 
+      pin: 3
```

Comment:
This is a bit odd, that we are accessing GPIO4 but calling the pin number 3
<img width="609" alt="image" src="https://github.com/user-attachments/assets/23b5a956-ff97-4c66-851c-a0a3f79e7f1d">

Reply:
yes sure, that is only because of the confusion with table 114 in the data sheet.  I will fix this! Thanks for manually testing the pin configuration!